modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -290,7 +298,9 @@ def merge_clinic(appt)
       end
 
       def merge_facility(appt)
-        appt[:location] = mobile_facility_service.get_facility(appt[:location_id]) unless appt[:location_id].nil?
+        return if appt[:location_id].nil?
+
+        appt[:location] = mobile_facility_service.get_facility(appt[:location_id]) || FACILITY_ERROR_MSG
```

Comment:
Updating this to set the error message like how the function behaved in the controller.